apps/src/lib/script-editor/ScriptEditor.jsx

```diff
@@ -51,12 +51,18 @@ const ScriptEditor = React.createClass({
     hasLessonPlan: PropTypes.bool,
     curriculumPath: PropTypes.string,
     announcements: PropTypes.arrayOf(announcementShape),
+    supportedLocales: PropTypes.arrayOf(PropTypes.string),
+    locales: PropTypes.arrayOf(PropTypes.arrayOf(PropTypes.string)).isRequired,
   },
 
   handleClearProjectWidgetSelectClick() {
     $(this.projectWidgetSelect).children('option')['removeAttr']('selected', true);
   },
 
+  handleClearSupportedLocalesSelectClick() {
+    $(this.supportedLocaleSelect).children('option')['removeAttr']('selected', true);
```

Comment:
I don't love blending jquery and react. Why aren't the `option`s simply rendered as selected or not based on the components state?

Reply:
Good question.  I was just using the same technique as the code immediately above.  Would prefer to punt for this PR if that's okay.